devel/hg-evolve/Portfile

```diff
@@ -45,3 +43,8 @@ evolve =
 post-patch {
     file delete ${worksrcpath}/hgext3rd/__init__.py
 }
+
+test {
+    file copy ${prefix}/share/mercurial/contrib/run-tests.py ${worksrcpath}/tests
+    system -W ${worksrcpath} "${python.bin} ./tests/run-tests.py -vj ${build.jobs}"
+}
```

Comment:
does this actually allow to run the tests? I would think you should at least set `test.run yes` and then ideally use the typical `test.cmd` and related things to run the actual tests instead of doing a system call.